subsys/bluetooth/mesh/Kconfig.dk_prov

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig BT_MESH_DK_PROV
+	bool "Bluetooth Mesh Provisioning handler for Nordic DKs"
+	depends on HWINFO
+	help
+	  Basic implementation of a Bluetooth Mesh Provisioning handler for the
+	  nRF5x development kits. Configures the UUID and handles some basic OOB
+	  methods. Used in samples as a reference implementation for
+	  provisioning handlers.
+
+if BT_MESH_DK_PROV
+
+config BT_MESH_DK_PROV_OOB_BLINK
+	bool "Enable blink output OOB method"
+	depends on DK_LIBRARY
```

Comment:
Use select instead?

Reply:
From a usage perspective, I agree, but according to [the Kconfig spec](https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt), we shouldn't really use select for this type of stuff. @SebastianBoe any general opinions on usage of select?

Reply:
They both have their problems unfortunately, in this case I would prefer to use depends, but it's not clear-cut.